articles/application-gateway/application-gateway-backend-health-troubleshooting.md

```diff
@@ -442,6 +442,8 @@ This behavior can occur for one or more of the following reasons:
 
 1.	To verify that Application Gateway is healthy and running, go to the **Resource Health** option in the portal and verify that the state is **Healthy**. If you see an **Unhealthy** or **Degraded** state, [contact support](https://azure.microsoft.com/support/options/).
 
+    a. For more information on resource health status, see [Azure Application Gateway Resource Health overview](https://docs.microsoft.com/en-us/azure/application-gateway/resource-health-overview)
```

Comment:
Can you remove /en-us/ from the link? We need to globalize hyperlinks to get it merged.

Reply:
Done.